test/functional/interface_rest.py

```diff
@@ -440,5 +440,20 @@ def run_test(self):
         resp = self.test_rest_request(f"/deploymentinfo/{INVALID_PARAM}", ret_type=RetType.OBJ, status=400)
         assert_equal(resp.read().decode('utf-8').rstrip(), f"Invalid hash: {INVALID_PARAM}")
 
+        self.log.info("Test the /broadcast URI")
+        tx = self.wallet.create_self_transfer()
+        resp_hex = self.test_rest_request("/broadcast", http_method='POST', req_type=ReqType.HEX, body=tx["hex"], ret_type=RetType.OBJ)
+        assert_equal(resp_hex.read().decode('utf-8').rstrip(), tx["txid"])
+        self.sync_all()
```

Comment:
In dff67a815c2780ce5328d65a77508d29d606c364 "rest: Add "/broadcast" endpoint": You're using the node0 to create and broadcast the transaction. Then, you sync the nodes and check if the transaction is into node0's mempool. Shouldn't you check if the transaction is into node1's mempool to ensure the broadcast worked?

Reply:
Oh yes, thank you for spotting this! Fixed in 1e07530836fad9899aecb45f4676e768f253fb25